<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>741</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器</string>
   </property>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dset">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>动态创建单次定时器</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>69</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>定时周期</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_setinterval">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="suffix">
     <string>ms</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_contintue">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连续定时</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_singal">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>单次定时</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>741</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器精度</string>
   </property>
   <widget class="QRadioButton" name="radioButton_PreciseTimer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PreciseTimer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_CoarseTimer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CoarseTimer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VeryCoarseTimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>741</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>当前时间（小时：分：秒）</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="lcdNumber_hour">
       <property name="value" stdset="0">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_mintus_">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>5</number>
       </property>
       <property name="value" stdset="0">
        <double>23.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_second">
       <property name="value" stdset="0">
        <double>6.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>741</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
